website/source/guides/operations/deployment-guide.html.md

```diff
@@ -40,7 +40,7 @@ Precompiled Nomad binaries are available for download at [https://releases.hashi
 You should perform checksum verification of the zip packages using the SHA256SUMS and SHA256SUMS.sig files available for the specific release version. HashiCorp provides [a guide on checksum verification](https://www.hashicorp.com/security.html) for precompiled binaries.
 
 ```text
-NOMAD_VERSION="0.8.4"
+export NOMAD_VERSION="0.8.7"
 curl --silent --remote-name https://releases.hashicorp.com/nomad/${NOMAD_VERSION}/nomad_${NOMAD_VERSION}_linux_amd64.zip
 curl --silent --remote-name https://releases.hashicorp.com/nomad/${NOMAD_VERSION}/nomad_${NOMAD_VERSION}_SHA256SUMS
 curl --silent --remote-name https://releases.hashicorp.com/nomad/${NOMAD_VERSION}/nomad_${NOMAD_VERSION}_SHA256SUMS.sig
```

Comment:
This guide downloads checksums and signature yet doesn't verify them.  I would suggest having another code block for downloading signatures and verifying them; or skipping these with a more explicit call out to the the checksum verification guide linked above.

Reply:
I just removed it :( See description on the commit: https://github.com/hashicorp/nomad/pull/5443/commits/673a168b7258b4ced3442b6128998e5b9391bc0b

Let me know if you feel like I should try harder to include it or at least more strongly encourage users to verify downloads. It's just such an awkward process and extremely difficult to fully explain.